src/types/responses/Block.ts

```diff
@@ -28,3 +30,16 @@ interface ILog {
 	index: number;
 	value: Codec;
 }
+
+export interface ICalcFee {
+	calcFee:
+		| {
+				calc_fee: () => null;
+		  }
+		| CalcFee
+		| undefined;
+	specName: string | number;
+	specVersion: string | number;
+	decorated: DecoratedMeta | undefined;
+	runtimeDoesNotMatch: boolean | undefined;
```

Comment:
```suggestion
	decorated?: DecoratedMeta;
	runtimeDoesNotMatch?: boolean;
```

I read in the typescript docs a while back that this is preffered.